README.md

```diff
@@ -173,7 +173,7 @@ element may be broken up into multiple chunks.
 This class reassembles these elements by buffering incomplete ones.
 
 The `Decoder` supports the same optional parameters as the underlying
-[`str_getcsv()`](http://php.net/str_getcsv) function.
+[`str_getcsv()`]((https://www.php.net/manual/en/function.str-getcsv.ph) function.
```

Comment:
This doesn't look right:

```suggestion
[`str_getcsv()`](https://www.php.net/manual/en/function.str-getcsv.php) function.
```

Reply:
Opps, I copied it wrong, thank you for noticing this @clue. Should be fixed now. 👍